docs/book/user-guide/cloud-guide/aws-guide.md

```diff
@@ -0,0 +1,201 @@
+```
+
+Once this is done, you can create the ZenML stack component as follows:
+
+2. Register a Sagemaker Pipelines orchestrator stack component:
+
+```shell
+zenml orchestrator register sagemaker-orchestrator --flavor=sagemaker --region=<REGION>
+```
+
+**Note**: SageMaker orchestrator utilizes the AWS configuration for operation and does not require direct connection via a service connector for authentication, as it relies on your AWS CLI configurations or environment variables.
+
+More details [here](../../stacks-and-components/component-guide/orchestrators/sagemaker.md).
+
+### Container Registry (ECR)
+
+An [orchestrator](../production-guide/cloud-orchestration.md) is the compute backend to run your pipelines.
+
+1. Before you run anything within the ZenML CLI, head on over to AWS and create a Sagemaker domain (Skip this if you already have one)
+
+```shell
+aws ecr create-repository --repository-name zenml-repository --region <REGION>
+```
+
+Once this is done, you can create the ZenML stack component as follows:
+
+2. Register a ECR container registry stack component:
+
+```shell
+zenml container-registry register ecr-registry --flavor=aws --uri=<ACCOUNT_ID>.dkr.ecr.<REGION>.amazonaws.com --connector aws-connector
+```
+
+More details [here](../../stacks-and-components/component-guide/container-registries/aws.md).
+
+## 4) Create stack
+
+{% tabs %}
+{% tab title="CLI" %}
+```bash
+export STACK_NAME=aws_stack
+
+zenml stack register ${STACK_NAME} -o ${ORCHESTRATOR_NAME} \
+    -a ${ARTIFACT_STORE_NAME} -c ${CONTAINER_REGISTRY_NAME} --set
+```
+
+{% hint style="info" %}
+In case you want to also add any other stack components to this stack, feel free to do so.
+{% endhint %}
+{% endtab %}
+
+{% tab title="Dashboard" %}
+<figure><img src="../../../../.gitbook/assets/Create_Stack.png" alt=""><figcaption><p>Combine the three stack components and you have your AWS stack. Feel free to add any other component of your choice as well.</p></figcaption></figure>
+{% endtab %}
+{% endtabs %}
+
+## 5) And you're already done!
+
+Just like that, you now have a fully working GCP stack ready to go. Feel free to take it for a spin by running a pipeline on it.
+
+Define a ZenML pipeline:
+
+```python
+from zenml import pipeline, step
+
+@step
+def hello_world() -> str:
+    return "Hello from SageMaker!"
+
+@pipeline
+def aws_sagemaker_pipeline():
+    hello_step()
+
+if __name__ == "__main__":
+    aws_sagemaker_pipeline()
+```
+
+Save this code to `run.py` and execute it. The pipeline will use AWS S3 for artifact storage, Amazon SageMaker Pipelines for orchestration, and Amazon ECR for container registry.
+
+```shell
+python run.py
+```
+
+<figure><img src="../../.gitbook/assets/run_with_repository.png" alt=""><figcaption><p>Sequence of events that happen when running a pipeline on a remote stack with a code repository</p></figcaption></figure>
+
+Read more in the [production guide](../production-guide/production-guide.md).
```

Comment:
and say something here about where your run / pipeline will actually show up in sagemaker? (i.e. which region etc)